website/content/docs/concepts/domain-model/credential-libraries.mdx

```diff
@@ -7,39 +7,70 @@ description: |-
 
 # Credential Libraries
 
-A credential library is a resource
-that provides [credentials][]
-of the same type and same access level
-from a single [credential store][].
+A credential library is a resource that provides [credentials][] of the same type and same access level from a single [credential store][].
 
 ## Attributes
 
-A credential library has the following configurable attributes:
+A credential library has the following optional configurable attributes:
 
-- `name` - (optional)
-  If set, the `name` must be unique within the credential library's parent
-  [credential store][].
+- `name` - If you set this attribute, the `name` must be unique within the credential library's parent [credential store][].
 
-- `description` - (optional)
+- `description` - A user-defined description of the credential library for identification purposes.
```

Comment:
Looking at other pages in the domain model section, we are being explicit with the `(optional)` after each optional field. I think it would be good to keep that here to be consistent with the other docs.

Reply:
I added these back in for all the optional values.